frontend/src/v4/routes/components/textField/textField.component.tsx

```diff
@@ -167,14 +167,15 @@ export class TextField extends React.PureComponent<IProps, IState> {
 
 		if (requiredConfirm) {
 			this.setState({ currentValue: event.target.value });
-			field.onChange(event);
 
 			if (onBeforeConfirmChange) {
 				onBeforeConfirmChange(event);
 			}
 		} else if (onChange) {
 			onChange(event);
 		}
+
+		field.onChange(event);
```

Comment:
should this be moved out like this? I'm not super familiar with how this works @sanmont3drepo

Reply:
@sanmont3drepo  says it's ok!